OtripleS.Web.Api.Tests.Unit/Services/Foundations/Assignments/AssignmentServiceTests.Validations.Add.cs

```diff
@@ -76,10 +76,6 @@ public async void ShouldThrowValidationExceptionOnCreateWhenAssignmentIsNullAndL
                 key: nameof(Assignment.Content),
                 values: "Text is required");
 
-            invalidAssignmentException.AddData(
-                key: nameof(Assignment.Status),
-                values: "Value is invalid");
-
             invalidAssignmentException.AddData(
```

Comment:
How about a test for checking 'AssignmentStatus' is valid but not in Active status, so we have appropriate message around "Value is invalid"?

Reply:
This was the original status. But I'm not sure every `Add` mandates an active add. It could be a historical record that should not be active anymore. We're going to have to see the business flow for this before we make that level of restriction.